ui/partials/main/thActiveFiltersBar.html

```diff
@@ -10,8 +10,9 @@
             class="filtersbar-filter">
           <span title="Filter by {{ filter.field}}: {{ filter.value }}">
             <b>{{ filter.field }}:</b>
+            <span ng-if="filter.field === 'failure_classification_id'">{{ filter.text }}</span>
```

Comment:
I think it's worth showing the full length of the classification filter text. So no `| limitTo` here.